src/node.cc

```diff
@@ -261,12 +260,8 @@ MaybeLocal<Value> RunBootstrapping(Environment* env) {
 
   // Store primordials
   env->set_primordials(Object::New(isolate));
-  std::vector<Local<String>> primordials_params = {
-    env->primordials_string()
-  };
-  std::vector<Local<Value>> primordials_args = {
-    env->primordials()
-  };
+  std::vector<Local<String>> primordials_params = {env->primordials_string()};
+  std::vector<Local<Value>> primordials_args = {env->primordials()};
```

Comment:
There seems to be quite a bit of unrelated changes here?

Reply:
@addaleax - yes, `make format-cpp` did this! I don't know if there is a way to limit the formatting to the lines changed by the author. If it becomes difficult from review point of view, I can revert when I push next.

Reply:
You can pass a starting commit, so that it only affects your own changes:

```
CLANG_FORMAT_START=master make format-cpp
```

(If your `master` isn’t up to date, you might want to update it first.)